.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,37 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Has the bug already been reported?
+      description: "Please check that the bug you're reporting hasn't already been reported. You can use the GitHub issues search box to help with this."
+      options:
+        - label: "I have checked that the bug hasn't already been reported"
+          required: true
```

Comment:
They can't submit the issue until they check the checkbox that says that they've checked that the bug hasn't already been reported